### ISSUE TITLE
VNC in black screen

### ISSUE THREAD

**Issue by Capitasas** [modded-ubuntu/modded-ubuntu#134]
when i try to open the vnc its just a black screen after doing `vncstart`. this is the output of the comand vncstart `New Xtigervnc server 'modded-ubuntu' on port (my port) for display :1.
Use xtigervncviewer -SecurityTypes VncAuth -passwd /tmp/tigervnc._hgxwj/passwd :1 to connect to the VNC server.` it shows like there is a video output but there is only a black screen and a mouse.
![Screenshot_20231117_224534_RVNC Viewer](https://github.com/modded-ubuntu/modded-ubuntu/assets/101959772/b0d77f7b-476d-4a2b-9b40-28e991bf4757)

**Comment by BDhackers009** [comment]
did you ran `vncstart` as root?

**Comment by Capitasas** [comment]
So I need root or how I do that?

**Comment by Capitasas** [comment]
> did you ran `vncstart` as root?

So I need root?

**Comment by BDhackers009** [comment]
no. You should just run vncstart after login to ubuntu shell by typing ubuntu in the termux

**Comment by carlinhoshk** [comment]
I'm still getting black screen.
I ran the 2 bash scripts without any problems and then ran the vncstart command

**Comment by BDhackers009** [comment]
> I'm still getting black screen. I ran the 2 bash scripts without any problems and then ran the vncstart command

Please send screenshots of your problem

**Comment by BDhackers009** [comment]
> I have problem just like that

Please send screenshots of your problem. Show me how did you ran the GUI and connected to vnc

**Comment by SUMITDEY69** [comment]
That happened to me, just type "vncserver" , it'll start. And if you want to adjust it to your display size then type the following ->
    vncserver -geometry 1920x1080

Here, 1920x1080 is the resolution that I require, yours maybe different, set the resolution according to your preference.

**Comment by BDhackers009** [comment]
Ig the issue is fixed
